charts/opensearch/values.yaml

```diff
@@ -323,12 +323,22 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+clusterHealthCheckParams: "wait_for_status=green&timeout=1s"
+
+startupProbe:
+  enabled: false
+  tcpSocket:
+    port: 9200
+  initialDelaySeconds: 5
+  periodSeconds: 10
+  timeoutSeconds: 3
+  failureThreshold: 30
 readinessProbe:
+  tcpSocket:
+    port: 9200
+  periodSeconds: 5
```

Comment:
Any particular reason why this has been halved from the default of 10 seconds?

Reply:
There is no solid reason, just some hit and trial experiments.